src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task{
+    private String by;
+
+    public Deadline(String description,String by){
```

Comment:
Inconsistance spacing  should be this "public Deadline(String description, String by){ "